### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_VS Code FAQ_Link Purpose: Link purpose is not unique and descriptive for all the links

### ISSUE THREAD

**Issue by Amlan454** [microsoft/vscode-docs#2668]
Check out Keros!- Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
code.visualstudio.com/docs/supporting/faq
Browser: chrome, edge
screen Reader: Narrator, NVDA
MAS violated: MAS2.4.4

**Note: This issue is observed in below pages of the application:**
 code.visualstudio.com/docs/supporting/faq
 code.visualstudio.com/docs/setup/linux
 code.visualstudio.com/docs/python/python-tutorial
 code.visualstudio.com/updates/v1_30
 code.visualstudio.com/updates/v1_29
 code.visualstudio.com/docs/editor/integrated-terminal
 code.visualstudio.com/docs/editor/debugging
 code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:**
1)Hit the URL(code.visualstudio.com/docs/supporting/faq)and login by using appropriate credential
2)Navigate to any of the link and check whether it is descriptive or not

**Actual Result:**
Link purpose is not unique and descriptive for all the links present in the application

**Expected Result:**
Link should be unique and descriptive enough.
Ex: "Snap store" link is there which is not giving the proper description.
If screen reader will read "Visual Studio Code is officially distributed as a Snap package in the Snap Store:" 
then it will be more understandable for the user

**User Impact:**
Screen reader users will not understand the purpose of the link , if screen reader will not announce the whole description about the link

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Comment by Amlan454** [comment]
![Bug_2668_link purpose is not descriptive](https://user-images.githubusercontent.com/42638683/57509362-ae7c2f80-7321-11e9-8c55-cc60cf0ed98f.png)

**Comment by Amlan454** [comment]
![Bug_2668_1](https://user-images.githubusercontent.com/42638683/57600445-75d59380-7577-11e9-9b25-99c44753d66a.png)

**Comment by Amlan454** [comment]
![Bug_2668_2](https://user-images.githubusercontent.com/42638683/57600608-ed0b2780-7577-11e9-889a-9ff73d180a6e.png)

**Comment by gregvanl** [comment]
Updated link text with https://github.com/microsoft/vscode-docs/commit/96c5ec9d3064c838306aba20653114873703fc07. Specifically reviewed and updated [here]() link text.
Given that the MAS 2.4.4 is Link Purpose (In Context), I think some of the flagged links are fine. For example, [Snap Store] is accurate and descriptive for a Linux user.

**Comment by mstechie** [comment]
vscode-docs_May2019
